src/source/Ice/IceAgent.h

```diff
@@ -104,6 +105,9 @@ typedef struct {
      * TurnConnectionTracker this candidate is associated to */
     PTurnConnectionTracker pTurnConnectionTracker;
 
+    /* If candidate is local. Indicate whether candidate
+     * has been reported through IceNewLocalCandidateFunc */
+    BOOL reported;
```

Comment:
I see this is calloc-ed so it's false. Just want to make sure that I checked all of the locations